assets/src/components/detours/detourMap.tsx

```diff
@@ -125,20 +132,29 @@ export const DetourMap = ({
     dispatch,
   ] = useContext(StateDispatchContext)
 
+  const [streetViewEnabled, setStreetViewEnabled] = useState(false)
+
   return (
     <div
       className={joinClasses([
         "h-100",
-        onClickMap && "c-detour_map--map__clickable",
+        !streetViewEnabled && onClickMap && "c-detour_map--map__clickable",
```

Comment:
I am _very_ much not a fan of this boolean goop. But this worked, I have high confidence in it, and I don't want the fix to be blocked on a refactor, which is why I PR'ed it as-is.

The refactor I'm thinking is to have `onPlaceWaypoint` be an input directly into `DetourMap` and then have `onClickMap` be computed based on `streetViewEnabled` and the rest of the detour state.